packages/ast/test/snapshots/valid/namespaces/output.js

```diff
@@ -3,7 +3,10 @@ module.exports = {
     type: "XMLDocument",
     rootElement: {
       type: "XMLElement",
-      namespaces: [{ prefix: "f", uri: "https://blah.com/furniture" }],
+      namespaces: [
+        { prefix: "f", uri: "https://blah.com/furniture" },
+        { prefix: undefined, uri: "https://blah.com/default" }
```

Comment:
should we make it explicitly empty? as in `""`?

Reply:
We could, but I think in think having `undefined` is better, because then we can have different AST for `<e xmlns:=""/>` and `<e xmlns=""/>`